docs/posts/introduction-to-ballerina-0.970.md

```diff
@@ -0,0 +1,220 @@
+
+
+## Control Flow Statements
+
+### If/Else Statement
+An if/else statement provides a way to perform conditional execution. It contains three sections: an if block, followed by multiple else if blocks, and finally a single else block. All the else if blocks and the else blocks are optional. Any number of statements can be defined inside each of these blocks.
+
+### Match Statement
+A match statement is a type switching construct that allows selective code execution based on the type of the expression that is being tested. The match statement can have one or more patterns with a type associated to it. Each pattern have statements that will get executed if that type is matched.
+
+### While
+The while looping construct will iterate and execute the code block within the while block continuously, until the condition for while loop is true.
+
+### Foreach
+The foreach looping construct will traverse through the items of a collection of data such as arrays, maps, JSON, and XML and execute the given code block.
+
+With the above two looping constructs, statements such as break and next can also be used, where the break statement would end the loop and next statement would go to the next iteration in the loop.
+
+### Iterable Operations
+Iterable operations can be used with types such as array, map, json, table, and xml. 
+
+## Transaction
+A Ballerina transaction is a series of data manipulation statements that must either fully complete or fully fail, and thereby leave the system in a consistent state. The transaction block is used handle Ballerina transactions. Any transaction aware operations performed within the transaction block are automatically committed or rolled back at the end of the block. Within transaction block, abort or retry statements can be used to explicitly force to abort the transaction or retry the transaction. 
+Ballerina supports Local Transactions, XA Transactions, Distributed Transactions. 
+
+
+## Error Handling
+The Ballerina approach of error handling is to introduce a first-class error concept that can both be returned as yet another return value (and thereby processed by the caller as it deems fit) or be thrown.
+
+The try-catch-finally blocks are basically used for catching and handling exceptions thrown during execution. The error lifting operator allows to walk down a set of fields of an object or tuple, without worrying whether there will be a error or null along the way. 
+check is a unary expression that is used to handle errors. The check expression removes the error type from the result of the sub expression and handle error case separately. 
+
+
+## Concurrency Constructs
+The concurrency model is based on workers, which are the fundamental execution units defined in Ballerina. The worker model is designed to be very light-weight constructs, which follows a fully non-blocking approach in its executions, which in-turn makes sure it has optimal utilization of the CPU.
```

Comment:
Change to 'The worker model is designed to be very light-weight constructs that follow a fully non-blocking approach in their executions. This in-turn makes sure it has optimal utilization of the CPU.'